README.md

```diff
@@ -114,14 +111,20 @@ task-02.02   2015-09-04   2eb21d8f9694146deca8c69cbc4a82acd62d395f   success
 
 ```
 
-`pytest` runner
----------------
+## `pytest` runner
+
+Many of the specs have python script to run the tests automatically. By default
+experiments will be launched on [IoT-LAB saclay site] since it has most of the
+boards used in the release-specs. It can also be ran locally as long as the
```

Comment:
[…] if not run on `native`?

Reply:
Mhh these sentences are still confusing though. Some tests are run on the IoT-LAB saclay site, some are run on `native`. Those who run on the IoT-LAB saclay site can also be run locally. This is a bit different from your original design where `native` nodes were only run in the "only local" setup. How about changing this whole paragraph to something like this

```md
Many of the specs have python script to run the tests automatically. By default
experiments will be launched on [IoT-LAB saclay site] that require non-`native`
boards. The saclay site was chosen since it has most of the non-`native` boards
used in the release-specs. Tests running on `native` will always be run locally
on the executing machine.  Non-`native` can also be ran locally as long as the
required `BOARDs` are available (see [local requirements](#local-requirements)).
```